src/main/scala/nl.knaw.dans.easy.license/package.scala

```diff
@@ -15,8 +15,20 @@
  */
 package nl.knaw.dans.easy
 
+import scala.util.{ Failure, Success, Try }
+
 package object license {
 
   type DatasetID = String
   type DepositorID = String
+
+  implicit class TryExtensions2[T](val t: Try[T]) extends AnyVal {
```

Comment:
you copied this from somewhere. Please state the source, or add it to the library itself.